### PR TITLE
2.43.0-rc.1

### PR DESCRIPTION
**Pull request by roidelapluie**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.43.0-rc.1 / 2023-03-16
+
+* [BUGFIX] Fixed a bug where changes in the Builder's Range function could lead to skipped labels during iteration (#12145)
```
Comment (bboreham) [inline CHANGELOG.md:5]: Might be worth mentioning this is a fix to RC0; the code did not exist in 2.42.

**Review by bboreham — APPROVED** [review]
lgtm; one thought about the changelog

**Review by Nexucis — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (roidelapluie) [reply CHANGELOG.md:5]: This is usually the case with rc.1 and it will not appear in the 2.42.0 final changelog.

**Pull request merged**